### ISSUE TITLE
1.12.0 fails to start after install

### ISSUE THREAD

**Issue by MHacker9404** [rancher-sandbox/rancher-desktop#6295]
### Actual Behavior

RD fails to start after install - just spins on updating certificates

### Steps to Reproduce

Win11 Pro with WSL2 latest version
Install RD 1.12.0 with AllUsers
Reboot host
Start RD

### Result

nothing

### Expected Behavior

RD runs

### Rancher Desktop Version

1.12.0

### Rancher Desktop K8s Version

0

### Which container engine are you using?

containerd (nerdctl)

### What operating system are you using?

Windows

### Operating System / Build Version

11 Pro 22H2 22621 .3007

### What CPU architecture are you using?

x64

### Linux only: what package format did you use to install Rancher Desktop?

None

### Windows User Only

BitDefender anti-virus

**Comment by MHacker9404** [comment]
[networking-ca.log](https://github.com/rancher-sandbox/rancher-desktop/files/13889462/networking-ca.log)
[dashboardServer.log](https://github.com/rancher-sandbox/rancher-desktop/files/13889463/dashboardServer.log)
[k8s.log](https://github.com/rancher-sandbox/rancher-desktop/files/13889464/k8s.log)
[background.log](https://github.com/rancher-sandbox/rancher-desktop/files/13889465/background.log)
[networking.log](https://github.com/rancher-sandbox/rancher-desktop/files/13889467/networking.log)

**Comment by mook-as** [comment]
`networking-ca.log` says:

```
unexpected fault address 0x230de7af210
fatal error: fault
[signal 0xc0000005 code=0x0 addr=0x230de7af210 pc=0xad3bba]

goroutine 1 [running]:
runtime.throw({0x1021937?, 0x40?})
	C:/hostedtoolcache/windows/go/1.21.5/x64/src/runtime/panic.go:1077 +0x65 fp=0xc00058f9d0 sp=0xc00058f9a0 pc=0xa5afe5
runtime.sigpanic()
	C:/hostedtoolcache/windows/go/1.21.5/x64/src/runtime/signal_windows.go:377 +0xd0 fp=0xc00058fa18 sp=0xc00058f9d0 pc=0xa6eef0
encoding/base64.(*Encoding).Encode(0xc0000c78c0?, {0xc0000c2db8?, 0x400?, 0x400?}, {0x230de7af210?, 0x480?, 0x15ad1a0?})
	C:/hostedtoolcache/windows/go/1.21.5/x64/src/encoding/base64/base64.go:156 +0x7a fp=0xc00058fa38 sp=0xc00058fa18 pc=0xad3bba
encoding/base64.(*encoder).Write(0xc0000c2d80, {0x230de7aef10?, 0x11?, 0x1021c21?})
	C:/hostedtoolcache/windows/go/1.21.5/x64/src/encoding/base64/base64.go:241 +0x10f fp=0xc00058fab8 sp=0xc00058fa38 pc=0xad400f
encoding/pem.Encode({0x120b6e0?, 0xc00010c020}, 0xc00058fce0)
	C:/hostedtoolcache/windows/go/1.21.5/x64/src/encoding/pem/pem.go:292 +0x357 fp=0xc00058fc70 sp=0xc00058fab8 pc=0xae2557
```
That's https://github.com/golang/go/blob/go1.21.5/src/encoding/base64/base64.go#L156 which probably means `src` is invalid.

https://github.com/rancher-sandbox/rancher-desktop/blob/de85021aeb0a24dc7be1fddd4db9dd96b40d6a1f/src/go/wsl-helper/cmd/certificates_windows.go#L54
`block` is probably invalid, then?
It basically comes from https://github.com/rancher-sandbox/rancher-desktop/blob/de85021aeb0a24dc7be1fddd4db9dd96b40d6a1f/src/go/wsl-helper/pkg/certificates/certificates_windows.go#L77

We may need to take a copy of that slice; since we're passing it over a channel it's possible that `x509.ParseCertificate` isn't making copies as I expected.

**Comment by fm-knopfler** [comment]
There might be different behaviour when running RD as admin vs non-admin.

The following appears in background.log when running as common user:

```
2024-01-11T09:01:36.123Z: Background process Vtunnel Host Process (pid 9924) exited with status 1 signal null
2024-01-11T09:01:37.234Z: Launching background process Vtunnel Host Process.
2024-01-11T09:01:43.134Z: Background process Win32 socket proxy (pid 1376) exited with status 1 signal null
2024-01-11T09:01:44.147Z: Launching background process Win32 socket proxy.
2024-01-11T09:01:48.729Z: Background process Vtunnel Host Process (pid 24820) exited with status 1 signal null
```

When launching as admin, it gets stuck on ca certificates.

In my case, running it as admin, quitting RD and start it as a normal user does the trick. Just launching as a non-privileged user in the first place doesn't work, see above.

**Comment by MHacker9404** [comment]
That worked the 1st time - but this morning when I started RD - it's just spinning on `installing CA certificates`
![image](https://github.com/rancher-sandbox/rancher-desktop/assets/4184348/5876bec1-ef48-4e64-92e7-037bd9c7fd09)
[vtunnel-host.log](https://github.com/rancher-sandbox/rancher-desktop/files/13903364/vtunnel-host.log)
[wsl-init.log](https://github.com/rancher-sandbox/rancher-desktop/files/13903365/wsl-init.log)
[dashboardServer.log](https://github.com/rancher-sandbox/rancher-desktop/files/13903366/dashboardServer.log)
[host-resolver-host.log](https://github.com/rancher-sandbox/rancher-desktop/files/13903367/host-resolver-host.log)
[k8s.log](https://github.com/rancher-sandbox/rancher-desktop/files/13903368/k8s.log)
[update.log](https://github.com/rancher-sandbox/rancher-desktop/files/13903369/update.log)
[diagnostics.log](https://github.com/rancher-sandbox/rancher-desktop/files/13903370/diagnostics.log)
[settings.log](https://github.com/rancher-sandbox/rancher-desktop/files/13903371/settings.log)
[wsl-exec.log](https://github.com/rancher-sandbox/rancher-desktop/files/13903372/wsl-exec.log)
[networking-ca.log](https://github.com/rancher-sandbox/rancher-desktop/files/13903373/networking-ca.log)
[background.log](https://github.com/rancher-sandbox/rancher-desktop/files/13903374/background.log)
[server.log](https://github.com/rancher-sandbox/rancher-desktop/files/13903375/server.log)
[shortcuts.log](https://github.com/rancher-sandbox/rancher-desktop/files/13903376/shortcuts.log)
[wsl.log](https://github.com/rancher-sandbox/rancher-desktop/files/13903377/wsl.log)
[integrations.log](https://github.com/rancher-sandbox/rancher-desktop/files/13903378/integrations.log)

**Comment by MHacker9404** [comment]
so that may have been the networking tunnel - turning that on now spinning on `updating kubeconfig`
![image](https://github.com/rancher-sandbox/rancher-desktop/assets/4184348/6da1e994-5185-4387-a5f0-d513e27c1588)
[wsl-exec.log](https://github.com/rancher-sandbox/rancher-desktop/files/13904043/wsl-exec.log)
[k3s.log](https://github.com/rancher-sandbox/rancher-desktop/files/13904044/k3s.log)
[shortcuts.log](https://github.com/rancher-sandbox/rancher-desktop/files/13904045/shortcuts.log)
[background.log](https://github.com/rancher-sandbox/rancher-desktop/files/13904046/background.log)
[wsl.log](https://github.com/rancher-sandbox/rancher-desktop/files/13904047/wsl.log)
[wsl-init.log](https://github.com/rancher-sandbox/rancher-desktop/files/13904048/wsl-init.log)
[diagnostics.log](https://github.com/rancher-sandbox/rancher-desktop/files/13904049/diagnostics.log)
[host-switch.log](https://github.com/rancher-sandbox/rancher-desktop/files/13904050/host-switch.log)
[dashboardServer.log](https://github.com/rancher-sandbox/rancher-desktop/files/13904051/dashboardServer.log)
[integrations.log](https://github.com/rancher-sandbox/rancher-desktop/files/13904052/integrations.log)
[k8s.log](https://github.com/rancher-sandbox/rancher-desktop/files/13904053/k8s.log)
[update.log](https://github.com/rancher-sandbox/rancher-desktop/files/13904054/update.log)
[server.log](https://github.com/rancher-sandbox/rancher-desktop/files/13904055/server.log)
[settings.log](https://github.com/rancher-sandbox/rancher-desktop/files/13904056/settings.log)

**Comment by jandubois** [comment]
This has been fixed in #6308 and will be included in a 1.12.1 patch release.
